templates/common/packages/config/endpoints.js

```diff
@@ -14,3 +14,16 @@ export const endpoints = {
  */
 export const contentDomain = 'https://jsonplaceholder.typicode.com/';
 export const apiGeeDomain = 'https://jsonplaceholder.typicode.com/';
+
+export const apiEndPoints = {
```

Comment:
This endpoint structure isn't scalable; this will always require code changes when you want to switch from mock to real or vice-versa. 

We should follow structure we have on line 15 and 16. Where you are defining your Api origin and service path (endpoint) differently. 

Now when someone switches to mock vs real API they should only be making origin switch. Endpoint should stay same.